Makefile

```diff
@@ -40,6 +40,8 @@ SSH_PROXY=-L6000:localhost:6000
 SSH_KEY=$(CONF_DIR)/ssh.key
 # Use QEMU H/W accelearation (any non-empty value will trigger using it)
 ACCEL=
+# Use tpm (any non-empty value will trigger using it)
+TPM=
```

Comment:
Should we document this in the markdown file which has the examples of how to build and run?